tests/Microsoft.Identity.Web.Test/MicrosoftIdentityOptionsTests.cs

```diff
@@ -48,6 +48,7 @@ public void IsB2C_NullOrEmptyUserFlow_ReturnsFalse()
             Assert.False(options.IsB2C);
         }
 
+#pragma warning disable xUnit1012 // Null should only be used for nullable parameters
```

Comment:
Is it possible to use a nullable type instead of disabling the warning?

Reply:
I was worried that if the tests were testing for nulls changing to a nullable type would push the warning into the test and force us to handle the null there. This ensures the test functionality doesn't change.

Reply:
I think this is fair and am not super concerned about it especially since they're existing tests. Thanks!